src/App.css

```diff
@@ -48,14 +50,27 @@ main {
 
 .settings-page {
     width: 100%;
-    padding: 40px 20px 0px 40px;
+    flex: 1;
 }
 
 .settings-players {
     display: flex;
     flex-direction: column;
     width: 100%;
     gap: 13px;
+    padding: 40px 20px 0px 40px;
+}
+
+h2 {
+    font-size: 40px;
+    font-weight: 400;
+    font-family: "IM Fell English SC", serif;
```

Comment:
I wonder if it'd make more sense to create a classname that uses this font-family, then you can attach it to any elements you want, or at least make it consistent with all other headers. This could end up getting repeated otherwise (you already have it on the h2 and the h6)
```
h1, h2, h3, h4, h5, h6 {
    font-family: "IM Fell English SC", serif;
}
```

Reply:
Cleaned up and updated the css file. Altered the footer to be use more semantic tags.